dowhy/causal_refuters/placebo_treatment_refuter.py

```diff
@@ -22,6 +24,15 @@ class PlaceboTreatmentRefuter(CausalRefuter):
 
     :param random_state: The seed value to be added if we wish to repeat the same random behavior. If we want to repeat the same behavior we push the same seed in the psuedo-random generator.
     :type random_state: int, RandomState, optional
+
+    # using wording from joblib docs here since it's not that simple, depends on backend used
+    # btw should backend be another parameter?
+    # docs: https://joblib.readthedocs.io/en/latest/generated/joblib.Parallel.html
```

Comment:
Good point. How about having the basic description you have here and then linking to joblib docs for more info?

Can say in the general docstring (e.g., after line 17), For joblib-related params (n_jobs, verbose, backend), please refer to <link> for more details.

Reply:
That's a good idea!